config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -494,6 +494,7 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-apiversion-upgrade-main
+      # TODO: If we're removing the v1alpha4 jobs, do we still want to keep the api version upgrade job?
```

Comment:
yes I believe we want to keep that so we don't break upgrade for existing clusters with new versions

Reply:
@Jont828 can you please remove the TODO?